src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -3660,8 +3652,16 @@ static public ServiceConfig load(String name) throws IOException {
 
     ServiceConfig config = CodecUtils.readServiceConfig(filename);
     si.load(config);
+
+    if (runtime != null) {
+      runtime.invoke("publishConfigLoaded", name);

```

Comment:
also.. this isn't part of the service lifecycle interfaces.

Reply:
this is not part of the ServiceLifeCycleListener interface and should not be